pysat/instruments/champ_star.py

```diff
@@ -202,13 +205,13 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
 
     Parameters
     -----------
-    inst : (pysat.Instrument)
+    inst : pysat.Instrument
         Instrument class object, whose attribute clean_level is used to return
         the desired level of data selectivity.
 
     Returns
     --------
-    Void : (NoneType)
+    Void : NoneType
```

Comment:
Should the whole returns be removed?

Reply:
Looks like I missed this. On the road now, but feel free to update here or downstream